pkg/azurefile/nodeserver.go

```diff
@@ -593,3 +612,10 @@ func checkGidPresentInMountFlags(mountFlags []string) bool {
 	}
 	return false
 }
+
+func isClientIDEmpty(context map[string]string) bool {
+	if context[clientIDField] != "" || context[strings.ToLower(clientIDField)] != "" {
```

Comment:
this cannot cover `clientId`, it's better write like this, and write a ut for this function
```
	for k, v := range reqContext {
		switch strings.ToLower(k) {
  ...
```

Reply:
rename as `hasClientID`